docs/_data/translations.yml

```diff
@@ -47,3 +47,8 @@
   code: vi
   description: Bootstrap bằng tiếng Việt
   url: http://getbootstrap.com.vn
+
+  - name: Turkish
```

Comment:
Also, this entry should come after "Spanish" based on how we're alphabetizing these